snowflake/README.md

```diff
@@ -117,6 +117,19 @@ datadog-agent integration install datadog-snowflake==2.0.1
 
 3. [Restart the Agent][6].
 
+#### Collecting Organization Data
+
+By default, this integration monitors the `ACCOUNT_USAGE` schema, but can also monitor the `ORGANIZATION_USAGE` schema to monitor organization-level metrics.
+
+To collect organization metrics, you can create configure the integration by changing the schema field to `ORGANIZATION_USAGE` and increasing the `min_collection_interval` to reduce the number of queries to Snowflake, as most organization queries have a latency of up to 24 hours.
+
+Note: To monitor organization metrics, your `user` must have the `ORGADMIN` role.
+
+    ```yaml
+       - schema: ORGANIZATION_USAGE
+         min_collection_interval: 43200
+    ```
+
 #### Collecting data for multiple environments
 
 If you want to collect data for multiple Snowflake environments, add each environment as an instance in your `snowflake.d/conf.yaml` file. For example, if you needed to collect data for two users named `DATADOG_SYSADMIN` and `DATADOG_USER`:
```

Comment:
and add an example at the bottom of this section to collect both account and organization metrics

Reply:
Sure, I can do that!